sentry-kotlin-multiplatform/src/androidTest/kotlin/io/sentry/kotlin/multiplatform/SentryBreadcrumbTest.kt

```diff
@@ -0,0 +1,22 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.kotlin.multiplatform.extensions.toAndroidBreadcrumb
+import io.sentry.kotlin.multiplatform.extensions.toKMPSentryLevel
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class AndroidSentryBreadcrumbTest {
+
+    @Test
+    fun `convert SentryBreadcrumb to AndroidBreadcrumb has proper values`() {
```

Comment:
This test can be improved, such as both `level` is null most likely or the default value.
Its important to set values different than the default ones.